sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_multiple_node/language-modeling-multi-gpu-multi-node.ipynb

```diff
@@ -2,15 +2,15 @@
  "cells": [
```

Comment:
<p><a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3611/#Training-with-Native-PyTorch" target="_blank">Training with Native PyTorch</a> -&gt; <a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3611/#Training-with-Native-PyTorch" target="_blank">Training with Native PyTorch</a> + SM DDP</p><p> </p><p><a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3611/#Training-with-Optimized-PyTorch" target="_blank">Training with Optimized PyTorch</a> -&gt; <a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3611/#Training-with-Optimized-PyTorch" target="_blank">Training with </a>SageMaker Training Compiler</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Fix the markdown formatting.</p>